stubs/requests/requests/packages/__init__.pyi

```diff
@@ -1,3 +1,5 @@
+import urllib3 as urllib3
```

Comment:
This isn't enough to get import slike `from requests.packages.urllib3 import _collections` to work. It only defines `urllib3` as a symbol in the module `requests.packages`, without making it a submodule of `requests.packages`. For example, an import like this doesn't create `sys.modules["requests.packages.urllib3"]` at runtime. To get around this at runtime, some packages add an entry to `sys.modules` manually (e.g. `os.py` does `sys.modules['os.path'] = path`). That obviously won't work with type checkers.

We have a couple options:
1. Add an "incomplete" marker `__getattr__`. Type checkers special-case this: if it is present, you can import modules that don't even exist.
2. Keep the old file structure of `requests/packages/urllib3/foobar.pyi` files, but within each file, import from urllib3.

Option 2 is slightly closer to what `os.path` does. At runtime, `os.path` is created with `import someothermodule as path` in `os.py` followed by `sys.modules['os.path'] = path`, and the stubs have a corresponding `os/path.pyi` with a star import.

Reply:
Python's import system really is broken in so many subtle ways. Maybe we should just remove `requests.packages.urllib3` and update all imports. If someone really uses `requests.packages.urllib3` in their code, instead of `urllib3` (and doesn't want to update their imports), we can (or they) always update our stubs.

Reply:
This sounds good. Having stubs for `requests.packages.urllib3` has always felt weird to me, because they're really more of an implementation detail than something anyone should ever rely on.